modules/swift/examples/code_snippets/Getting-Started.swift

```diff
@@ -14,53 +24,36 @@ class ViewController: UIViewController {
         super.viewDidLoad()
         // Do any additional setup after loading the view.
 
-        getStarted(testReplication: false)
+        try! getStartedWithReplication(replication: false)
 }
 
-
-    func getStarted (testReplication: Bool) {
+    func getStartedWithReplication (replication: Bool) throws {
         // Get the database (and create it if it doesn’t exist).
-        let database: Database
-        do {
-            database = try Database(name: "mydb")
-        } catch {
-            fatalError("Error opening database")
-        }
+        let database = try Database(name: "mydb")
+        let collection = try database.defaultCollection()
 
         // Create a new document (i.e. a record) in the database.
-        let mutableDoc = MutableDocument()
+        var mutableDoc = MutableDocument()
             .setFloat(2.0, forKey: "version")
             .setString("SDK", forKey: "type")
 
-        // Save it to the database.
-        do {
-            try database.saveDocument(mutableDoc)
-        } catch {
-            fatalError("Error saving document")
-        }
+        // Save document to default collection.
+        try collection.save(document: mutableDoc)
         print("Created document id type \(mutableDoc.id)? with type = \(mutableDoc.string(forKey: "type")!)")
-
-
+        
         // Update a document.
-        if let mutableDoc = database.document(withID: mutableDoc.id)?.toMutable() {
-            mutableDoc.setString("Swift", forKey: "language")
-            do {
-                try database.saveDocument(mutableDoc)
-
-                let document = database.document(withID: mutableDoc.id)!
-                // Log the document ID (generated by the database)
-                // and properties
-                print("Updated document id \(document.id), adding language \(document.string(forKey: "language")!)")
-            } catch {
-                fatalError("Error updating document")
-            }
-        }
+        mutableDoc = try collection.document(id: mutableDoc.id)!.toMutable()
+        mutableDoc.setString("Swift", forKey: "language")
+        try collection.save(document: mutableDoc)
+        let document = try collection.document(id: mutableDoc.id)
+        assert(document!.string(forKey: "language") == "Swift",
```

Comment:
This is not the test so I don't think we should use assert here. Print info from the doc is fine.